### DISCUSSION TITLE
Re-activated admin pages disable from the Settings UI

### DISCUSSION THREAD

**Discussion by webmasterlocafe** [wp-graphql/wp-graphql#2942] [Q&A]
Hi team, simple question: How do I re-activated admin pages disable from the Settings UI by mistake:

[![Captura-de-Tela-2023-09-29-a-s-09-11-32.png](https://i.postimg.cc/7PW88PZV/Captura-de-Tela-2023-09-29-a-s-09-11-32.png)](https://postimg.cc/RqKbwBf6)

**Comment by jasonbahl** [comment]
@webmasterlocafe can you share more information on what steps you took to de-activate the settings ui? ~And the screenshot you shared is a 404 so we cannot see it.~

WPGraphQL by default doesn't have a setting to disable the admin settings page, so my hunch is that either another plugin is removing the WPGraphQL Settings page or the user you are logged into doesn't have proper capabilities to see the settings page. 

I would recommend testing in a clean install with no other plugins to ensure you can access the settings page. And then try activating other plugins one at a time until the settings page disappears and you might be able to identify another plugin that is hiding/removing the settings page.


----
edit: After initial reply, I was able to see the screenshot load. 

Based on the screenshot, it looks like at least WooGraphQL and WPGraphQL Cors are active. It might be worth trying to de-activate those to see if you see any difference in the rendering of the page? Perhaps a bug in one of the extending plugins is causing the settings to not render as expected?

Might also be worth testing with the WordPress locale set to English? Perhaps there's a bug when content is internationalized and we may need to investigate that if it can be consistently reproduced?
